src/isAnnotatedForRemoval.js

```diff
@@ -0,0 +1,7 @@
+// @flow weak
+
+export default function (node) {
+  const comments = node.trailingComments || [];
```

Comment:
I feel like this one is less intrusive than the other one. I would rather only support on for simplicity and consistency of the codebase. What do you think?